packages/expandable-card/src/ExpandableCard.tsx

```diff
@@ -264,12 +267,15 @@ const ExpandableCard = ({
 
         <Transition in={isOpen} timeout={transitionDuration}>
           {state => (
-            <Icon
+            <IconButton
+              // Setting 'as="div"' to avoid nesting interactive components for accessibility
+              as="div"
               className={cx(iconStyle, iconTransitionStyle[state])}
-              glyph="ChevronUp"
-              size={24}
-              fill={palette.gray.base}
-            />
+              aria-label={title}
+              tabIndex={0}
+            >
+              <Icon glyph="ChevronUp" size={24} fill={palette.gray.base} />
+            </IconButton>
```

Comment:
👍  Just played around with VoiceOver and I have just one nit: the `aria-label` should be something like "expand/collapse card". Having the title there again is a little confusing